msteams-platform/task-modules-and-cards/cards/cards-format.md

```diff
@@ -492,6 +492,219 @@ The following image shows the default view of the Adaptive Card when you have no
 
 :::image type="content" source="../../assets/images/Cards/small-width-adaptive-card.png" alt-text="Small width Adaptive Card view":::
 
+### Adaptive Card responsive layout
+
+Adaptive Cards usually match the look and feel of the host application and have a fixed layout that remains consistent across Teams mobile and desktop clients.
+
+Adaptive Card responsive layout helps you to design Adaptive Cards with different layouts that adapt to different card widths in the context where the card is displayed. When you design Adaptive Card for your app, ensure that you provide a better user experience across chat, channels, and meeting chat in Teams desktop and mobile clients.
+
+[Add image for responsive Adaptive card in mobile and desktop.]
+
+#### Design responsive Adaptive Cards
+
+You can design a responsive Adaptive Card using the `targetWidth` property, which allows you to show or hide any element based on the card's width.
+
+You can set the `targetWidth` property to any element in your Adaptive Card to create different layouts for different card widths. The following table provides the available values of the `targetWidth` property for different layouts:
+
+  |Value  |Description  |
+  |---------|---------|
+  | `veryNarrow` | Use for Adaptive Card on a desktop mode in a meeting chat. |
+  | `narrow` | Use for Adaptive Card on a mobile phone in portrait mode. |
+  | `standard` | Use for Adaptive Card  on a mobile phone in landscape mode, a tablet in portrait mode, or a chat in desktop mode. The default value is `standard`.|
+  | `wide` | Use for Adaptive Card on a tablet in landscape mode, or a chat or channels in desktop mode with the card width as `full width`. |
+
+The following images and JSON samples show an example of Adaptive Card designed without using `targetWidth` and using `targetWidth`:
+
+* **Adaptive Card designed without using `targetWidth`**.
+
+  <br> Adaptive Card with card width wide:</br>
```

Comment:
JSON should come before the screenshots.

Reply:
Please use this text:

**Card designed without using `targetWidth`**

JSON:
<include the card JSON here>

As rendered when the card is "wide", it looks good:
<screenshot of the wide card>

But look what happens the card's width is either "standard" or "narrow" - the name and role are starting to get squished pretty badly:
<screenshot of the standard/narrow card>

And when the card is "very narrow", it's even worse:
<screenshot of the very narrow card>

Let's fix it.

**The same card, but designed using `targetWidth`**

JSON:
<include the updated card JSON here>

As rendered when the card is "wide", it still looks good:
<screenshot of the wide card>

But now it also looks good when the card's width is either "standard" or "narrow" - the role has been moved to under the name as result of there not being enough horizontal space to show them side-by-side:
<screenshot of the standard/narrow card>

And when there's even less space, as is the case when the card is "very narrow", we can hide the image and only keep the most meaningful information (name and role):
<screenshot of the very narrow card>

For more information on how to design an Adaptive Card, [designing Adaptive Cards for your Microsoft Teams app](design-effective-cards.md).

Reply:
Updated